src/test/java/uk/gov/hmcts/reform/translate/errorhandling/RestExceptionHandlerTest.java

```diff
@@ -0,0 +1,220 @@
+    excludeFilters = @ComponentScan.Filter(type = ASSIGNABLE_TYPE, classes =
+        {SecurityConfiguration.class, JwtGrantedAuthoritiesConverter.class}))
+@AutoConfigureMockMvc(addFilters = false)
+@ImportAutoConfiguration(TestIdamConfiguration.class)
+class RestExceptionHandlerTest {
+
+    private static final String ERROR_PATH_ERROR = "$.error";
+    private static final String ERROR_PATH_MESSAGE = "$.message";
+    private static final String ERROR_PATH_STATUS = "$.status";
+
+    @Autowired
+    protected MockMvc mockMvc;
+
+    @MockBean
+    protected DictionaryService service;
+
+    @Autowired
+    protected ObjectMapper objectMapper;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
+    }
+
+    @DisplayName("should return correct response when generic ApiException is thrown")
+    @Test
+    void shouldReturnGenericApiExceptionResponse() throws Exception {
+
+        // GIVEN
+        String myUniqueExceptionMessage = "Generic Api Exception";
+        ApiException expectedException = new ApiException(myUniqueExceptionMessage);
+
+        /// WHEN
+        setupMockServiceToThrowException(expectedException);
+        ResultActions result = this.mockMvc.perform(get(DICTIONARY_URL)
+                                                        .contentType(MediaType.APPLICATION_JSON));
+
+        // THEN
+        assertHttpErrorResponse(result, HttpStatus.INTERNAL_SERVER_ERROR, expectedException.getMessage());
+
+    }
+
+    @DisplayName("should return correct response when RequestErrorException is thrown")
+    @Test
+    void shouldReturnRequestErrorExceptionResponse() throws Exception {
+
+        // GIVEN
+        RequestErrorException expectedException =
+            new RequestErrorException("invalidRole");
+
+        /// WHEN
+        setupMockServiceToThrowException(expectedException);
+        ResultActions result = this.mockMvc.perform(get(DICTIONARY_URL)
+                                                        .contentType(MediaType.APPLICATION_JSON));
+
+        // THEN
+        assertHttpErrorResponse(result, HttpStatus.FORBIDDEN, expectedException.getMessage());
+
+    }
+
+    @DisplayName("should return correct response when RoleMissingException is thrown")
+    @Test
+    void shouldReturnRoleMissingExceptionResponse() throws Exception {
+
+        // GIVEN
+        RoleMissingException expectedException =
+            new RoleMissingException(String.format(RoleMissingException.ERROR_MESSAGE, "test-role"));
+
+        /// WHEN
+        setupMockServiceToThrowException(expectedException);
+        ResultActions result = this.mockMvc.perform(get(DICTIONARY_URL)
+                                                        .contentType(MediaType.APPLICATION_JSON));
+
+        // THEN
+        assertHttpErrorResponse(result, HttpStatus.FORBIDDEN, expectedException.getMessage());
+
+    }
+
+    @DisplayName("should return correct response when CasePaymentOrdersFilterException is thrown")
```

Comment:
Can we reword

Reply:
Fixed in [f64e292](https://github.com/hmcts/ts-translation-service/pull/51/commits/f64e292233d45defe1b478426cae0b933d1bf0e6)